rllib/core/rl_module/encoder.py

```diff
@@ -40,7 +40,7 @@ def build(self):
 
 
 @dataclass
-class FCConfig(EncoderConfig):
```

Comment:
Shouldn't we just move the entirety of encoder.pys in rl_module to the experimental folder? To clean up the rl_module folder?